_includes/cockroachcloud/cockroachcloud-ask-admin.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-Only [Console Admins](console-access-management.html#console-admin) can create SQL users. If you are a [Developer](console-access-management.html#developer), you need to ask your Console Admin for the credentials of a SQL user to access the cluster. To find out who your Console Admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.
+Only [org admins](authorization.html#org-administrator-legacy) can create SQL users. If you are a [Developer](authorization.html#org-developer-legacy), you need to ask your org admin for the credentials of a SQL user to access the cluster. To find out who your org admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.
```

Comment:
```suggestion
Only [org admins](authorization.html#org-administrator-legacy) can create SQL users. If you are a [Developer](authorization.html#org-developer-legacy), you need to ask your org admin for the credentials of a SQL user to access the cluster. You can find the email of a cluster administrator for your cluster from the console's **Access** page, at `https://cockroachlabs.cloud/access`.
```

Reply:
Is it worth linking "Access page" to https://www.cockroachlabs.com/docs/cockroachcloud/console-access-management.html?

Reply:
That page has been removed, and it's content absorbed into managing-access.md